test/credentials_test.dart

```diff
@@ -284,12 +284,12 @@ Future<void> main([List<String>? args]) async {
     const String newPassword = "!@#!DQXQWD!223eda";
     final authProvider = EmailPasswordAuthProvider(app);
     await authProvider.registerUser(username, strongPassword);
-    await expectLater(() {
+    await expectLater(() async {
```

Comment:
@blagoev Why do we need `async` and `await` inside `expectLater`? It is not needed. I have also added them by mistake in some of my old tests, but I think I have to remove them.

Reply:
its not problematic per se. you can do it or not do it.